### ISSUE TITLE
Alpha 8.1 Compatibility

### ISSUE THREAD

**Issue by smudge202** [software-inc-modding/devbot#5]

**Comment by Skullex3** [comment]
I'm actually trying to update software for 8.2.2, but it take a lot of time :/

**Comment by smudge202** [comment]
Yup, I know that all too well I'm afraid. I'm actually putting my effort towards some tooling to make this stuff a **lot** easier. Do you do any .net work, @Skullex3 ?

**Comment by Skullex3** [comment]
Yeah, Atm I don't have any bugs from softwares, but now it's time for companies -.-

Sadly no, I wish I have time to learn .net, but I cant help you with it now :/

**Comment by smudge202** [comment]
Great job on the software. Feel free to push the changes separately so you can break it down into smaller chunks?

I'll create an alpha8.2.2 branch quickly.

**Comment by smudge202** [comment]
If you do a fetch from origin you should now see a `dev/alpha8-2-2` branch: https://github.com/software-inc-modding/devbot/branches

Target that when you get to the stage you're ready to pull request. :)

**Comment by pokmon987** [comment]
Okay, my question is whether or not you are still working on this at all... If not, I could take over. I got everything updated for 8.10.16, and wondering what I should do with the 3D and 2D editors and whether or not to combine the modded with the vanilla software. If you abandoned the project I can either stop working on it completely or I can take over for you until you want to work on it again. Once you do want to, you can have your godly powers over the modding community again.

**Comment by Skullex3** [comment]
Hey guys ! Got a mail notify, so finally I can give some news ^^ 

First, I'm sorry, I've been really busy with school, and until June I will not be able to help you. I followed the evolution of the game, and I think after my exam I will have a lot of time to help again. I'm really sorry, wanted to advise you several months ago, but I always forget ^^ 
This mod is a great idea, and I think we can make it better ;)

**Comment by pokmon987** [comment]
Yeah, It really is a good idea, basically adding on to the vanilla software, adding some more at the same time, and making it tiered. I updated everything myself to 8.10.16, but then I noticed some bugs, so I'll be working on that, and figuring out things for the 2D and 3D editors

**Comment by smudge202** [comment]
Sorry for the slow reply.

I'll be honest, I'm truly struggling to find any time at all to work on this. It's one of the reasons it's open source and why I placed the repo under the `software-inc-modding` organisation instead of using a person repository. Please do take the reigns. Anyone that can send a good pull request will be added as a collaborator to the repo (so you don't get held up waiting on me reviewing stuff).

I hope to one day find some time to come back and get everything back up to date, but it won't be today I'm afraid (along with work and home life, I'm currently writing [my own game](http://blog.devbot.net/clean-space-introduction/) so _free time_ is tricky! :smile:
